# A Predicted Wildfire raster

Header header
# Id of the WildfireMap this prediction is based on
string wildfire_id
# Id of this prediction
string prediction_id
# Last valid time for this prediction
time last_valid
Raster raster